src/modules/dmq_usrloc/usrloc_sync.c

```diff
@@ -825,6 +837,16 @@ int usrloc_dmq_send_contact(
 	srjson_AddNumberToObject(&jdoc, jdoc.root, "reg_id", ptr->reg_id);
 	srjson_AddNumberToObject(&jdoc, jdoc.root, "server_id", ptr->server_id);
 
+	/*  Loop through Χavp attributes of the contact and and create a json object */
+	srjson_t *jdoc_xavp = srjson_CreateObject(&jdoc);
```

Comment:
Variable should be declared at the beginning of the block to be coherent and also keep compliant with older C standards.

Reply:
Fixed on some other variables too on the newly static functions.